src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -195,7 +195,9 @@
    featured-communities
    view-type]
   (fn []
-    [rn/view {:padding-horizontal 20}
+    [rn/view
+     {:padding-horizontal 20
```

Comment:
Quick reminder: put the styles inside a `:style` map instead of mixing them with the rest of the component props. I added a note to update the guidelines with this convention.